.github/workflows/deployment-images.yaml

```diff
@@ -117,8 +117,6 @@ jobs:
           tags: ghcr.io/rentpath/${{ needs.set-deployment-image-name.outputs.name }}:${{ needs.set-version.outputs.version }}
           secrets: |
             "github_token=${{ secrets.GHCR_USER_PAT }}"
-          build-args: |
-            "GHCR_USER_PAT=${{ secrets.GHCR_USER_PAT }}"
```

Comment:
Dockerfiles now need to mount the `github_token` secret defined on line 119 instead.